src/app/dashboard-widgets/pipelines-widget/pipelines-widget.module.ts

```diff
@@ -15,6 +19,7 @@ import { PipelinesWidgetComponent } from './pipelines-widget.component';
     MomentModule
   ],
   declarations: [PipelinesWidgetComponent],
-  exports: [PipelinesWidgetComponent]
+  exports: [PipelinesWidgetComponent],
+  providers: [ PipelinesService ]
```

Comment:
Why did this need to be added here? The component was already using the PipelinesService before, wasn't it? In fact it appears to be providing its own PipelinesService. Should that have been removed?

Also, whitespace is inconsistent.

Reply:
I moved it here to make some tests work, because I was getting a complaint about the PipelinesService not being provided even though I had a mock in place. I've removed it from the component.

Reply:
Whitespace within the square brackets is still inconsistent